warehouse/utils/utils.go

```diff
@@ -305,6 +305,20 @@ type TriggerUploadRequest struct {
 	DestinationID string `json:"destination_id"`
 }
 
+type Connection struct {
+	SourceId      string   `json:"source_id"`
+	DestinationId string   `json:"destination_id"`
+	Tables        []string `json:"tables"`
+}
+
+type ConnectionsTablesRequest struct {
+	Connections []Connection `json:"connections"`
+}
```

Comment:
Let's not use Connection here it gives the impression that Tables are passed as requests but we are using it only for response.

Reply:
```GO

type SourceIDDestinationID struct {
	SourceID      string `json:"source_id"`
	DestinationID string `json:"destination_id"`
}

type ConnectionsTablesRequest struct {
	Connections []SourceIDDestinationID `json:"connections"`
}
``